data/json/items/gun/762.json

```diff
@@ -236,5 +236,78 @@
+    "price_postapoc": 4000,
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "vz58_p",
+        "name": { "str": "Samopal vz. 58 P" },
+        "description": "This lightweight and relatively svelte Cold War assault rifle was originally developed for use by the Czechoslovakian army, in place of the various licenced Kalashnikov rifles which were then prevalent around eastern Europe.  Baring more than a passing resemblance to the AK-47, this weapon nevertheless features a completely different array of internals and possesses no interchangeable parts with its Russian counterpart.  Making use of proprietary magazines and a 15.4 inch barrel, the vz. 58 was regarded by some to be the Kalashnikov’s lesser known superior.  Likely reconstructed from a pre-1986 imported parts kit, or brought back from a 3rd world war zone, this particular weapon features a removable wood impregnated plastic buttstock and a 'P' suffixed designation, which both mark it as an original infantry model.",
+        "weight": 1
+      },
+      {
+        "id": "vz58_v",
+        "name": { "str": "Samopal VZ 58 V" },
+        "description": "This lightweight and relatively svelte cold war assault rifle was originally developed for use by the Czechoslovak army, in place of the various licenced Kalashnikov rifles which were then prevalent around eastern Europe.  Baring more than a passing resemblance to the AK-47, this weapon nevertheless features a completely different array of internals and possesses no  interchangeable parts with its Russian counterpart.  Making use of proprietary magazines and a 15.4 inch barrel, the VZ. 58 is regarded by some to be the Kalashnikov’s lesser known superior.  Likely reconstructed from a pre-1986 imported parts kit, or brought back from a 3rd world war zone, this particular weapon features a removable side-folding buttstock and a 'V' suffixed designation, which both mark it as an original airborne or crew issued model."
+      }
+    ],
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic", "wood" ],
+    "color": "brown",
+    "ammo": [ "762" ],
+    "dispersion": 150,
+    "durability": 10,
+    "min_cycle_recoil": 1950,
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ],
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail mount", 1 ],
+      [ "sights mount", 1 ],
+      [ "sling", 1 ],
+      [ "stock", 1 ],
+      [ "stock accessory", 2 ],
+      [ "underbarrel mount", 1 ]
+    ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "vz58mag30", "vz58mag10" ] } ]
+  },
+  {
+    "id": "vz58ts",
+    "copy-from": "vz58_p",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "Czechpoint tactical rifle" },
```

Comment:
This should include `vz. 58`, or all the variants should be called this (and the magazines should have variants to use this).